HunterPie.Core/Game/World/Entities/MHWPlayer.cs

```diff
@@ -466,6 +469,10 @@ private void GetParty()
 
                 string name = _process.Memory.Read(playerAddress, 32);
                 bool isLocalPlayer = name == Name;
+                if (isLocalPlayer)
+                {
+                    localLocalPlayerAddress = playerAddress;
```

Comment:
Can't you just assign `_localPlayerAddress` here instead of creating a new variable for it?

Reply:
1. If we haven't discovered local player's party address, we want to keep `_localPlayerAddress` as zero.
2. The assignment to `_localPlayerAddress` here means I'm publicize this value to other members in this class.
 
If I just assign `_localPlayerAddress` here, there will be a short period of time when `_localPlayerAddress` is zero only because `GetParty` is still executing in progress.  I want to expose a consistent value to the other members of this class, instead of a variable that sometimes has value and sometimes is zero.